script.js

```diff
@@ -0,0 +1,64 @@
+$(document).ready(function(){
+    const movieGluAPI = '5zwPt8bzCu7r7rifnKoCd827XemsTLcO6rk51vTy';
+    // const NYTMovie = '';//we will need to grab the movie name from wherever it will be rendered on the page
+    // const NTYQueryMovies = 'https://api.nytimes.com/svc/movies/v2/reviews/search.json?query='+ NYTMovie + 'api-key=' + NYTAPI;
+    const TMDBAPI = '0adfd846cf8f1168484b5da4e5339d7c';
+    const personSearch = 'matthew mcconaughey' ;//function around get requests that gets value of input when search is clicked
```

Comment:
I know I just merged my branch into master, but it might be best to assign this the value of the input box